benchopt/plotting/html/templates/result.mako.html

```diff
@@ -91,30 +91,44 @@ <h1 class="ml11">
                             <label for="change_scaling" class="block text-sm font-medium text-white">Scale</label>
                             <div class="mt-1 rounded-md shadow-sm">
                                 <select id="change_scaling" class="focus:ring-blue-500 focus:border-blue-500 block w-full h-8 px-4 sm:text-sm border-gray-300 rounded-md" onchange="setState({scale: this.value})">
-                                    <option value="semilog-y">semilog-y</option>
-                                    <option value="semilog-x">semilog-x</option>
-                                    <option value="loglog">loglog</option>
                                     <option value="linear">linear</option>
+                                    <!-- Options for other plots -->
+                                    <option class="other_plot_option" value="semilog-y">semilog-y</option>
+                                    <option class="other_plot_option" value="semilog-x">semilog-x</option>
+                                    <option class="other_plot_option" value="loglog">loglog</option>
+                                    <!-- Option for boxplot -->
+                                    <option class="boxplot_option" value="log">log</option>
                                 </select>
                             </div>
                         </div>
                         <div id="xaxis-type-form-group" class="mt-8 px-8">
                             <label for="change_xaxis_type" class="block text-sm font-medium text-white">X-axis</label>
                             <div class="mt-1 rounded-md shadow-sm">
                                 <select id="change_xaxis_type" class="focus:ring-blue-500 focus:border-blue-500 block w-full h-8 px-4 sm:text-sm border-gray-300 rounded-md" onchange="setState({xaxis_type: this.value})">
-                                   <!-- options will be added dynamically -->
+                                    <option value="Time" class="hidden">Time</option>
+                                    <option value="Iteration" class="hidden">Iteration</option>
+                                    <option value="Solver" class="hidden">Solver</option>
                                 </select>
                             </div>
                         </div>
-                        <div class="flex items-center justify-between mt-8 px-8">
+                        <div id="yaxis-type-form-group" class="mt-8 px-8">
+                            <label for="change_yaxis_type" class="block text-sm font-medium text-white">Y-axis</label>
```

Comment:
We could put `runtime` as a value the same form as the `objective` column, that would be clearer I think?

We would put this option only for the box plot?